sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -550,8 +545,29 @@ class Dataset[T] private[sql](
       case ExplainMode.Formatted =>
         qe.simpleString(formatted = true)
     }
+  }
+
+  private[sql] def toExplainString(mode: String): String = {
```

Comment:
Is this only for Python to call? I think it is better to add short comment.

Reply:
Ah, yes. ok, I'll follow up, thanks for the comment.